### DISCUSSION TITLE
New Design UI - Background Color in with enabled navigation

### DISCUSSION THREAD

**Discussion by nico9312** [Budibase/budibase#6806] [Help] [answered]
Hi,
I updated one of my created apps to the new version and I also detatched the screen from the layout. After enabeling the navigation for the route the background of the screen is not applying to the whole screen. It would be nice to either specify the background color in the theme tab or to max out the screen to the full with and height when navbar is enabled

![image](https://user-images.githubusercontent.com/24295326/179732275-15ceabdf-abb2-42d5-8e6d-8eff9a15ef12.png)

**Comment by mmalessa80** [comment, marked as answer]
Can't quiet tell from your screenshot, but I would suspect that on the screen element you have margin (top, bottom, left, right) set to 12 or 16px maybe? If that is the case, the margin settings should be none, and the padding should be set instead.

**Reply by poirazis** [reply]
Hey @nico9312 
You can set the Screen Size to Max and that will cover the whole "canvas".
This way you can apply a background to the screen, and then insert a Container element inside the Screen to arrange your content. 

I agree it would be nice to be able to set the "canvas" background from the Theme tab, to avoid having to make this change to all the existing screens.

**Reply by nico9312** [reply]
Hey @poirazis,
Setting screen width to "max" solved my problem. Thank you a lot!

![image](https://user-images.githubusercontent.com/24295326/179956979-98ddc052-ca27-4da5-a772-96d0065c90e2.png)
